model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -128,18 +128,15 @@ class FindingsMatcher(
      */
     private fun matchFileFindings(
         licenses: List<LicenseFinding>,
-        copyrights: List<CopyrightFinding>,
-        rootLicenses: Collection<String>
+        copyrights: List<CopyrightFinding>
```

Comment:
I had several commit message nits, but it's easier to post the fixed message instead of listing them:

```
 FindingsMatcher: Discard less copyright statements

The current algorithm links copyright statements detected in files without
any detected license to all root license. At the same time copyright
statements detected in files with multiple detected licenses are discarded
if the license locations are not nearby. Instead, consistently assign such
copyright statements to all root licenses in order to move a bit from
under- towards overreporting which better matches ORT's philosophy.
As a result copyright statements are not discarded anymore in case there
is at least one root license detected.

I recall some historical reasoning behind that discarding, it said that
the likelihood of such discarded statements being false positive
detections is significantly higher. Whether this argument is sufficient
`to be legally on the safe side` is a matter of opinion, my tests with a
large scan indicated that most of the discarded copyrights statements
actually are not false positives.

Note: The ultimate goal should be to (optionally) not discard any
copyright statements at all. That is a bit harder to achieve which is why
this change is limited to that low hanging fruit.
```

Reply:
Thanks! Done.